machine_test.go

```diff
@@ -1958,6 +1958,55 @@ func TestLoadSnapshot(t *testing.T) {
 				require.NoError(t, err)
 			},
 		},
+		{
+			name: "TestLoadSnapshot without unnecessary config",
```

Comment:
Instead of a new test with minimum config I think it is better for readers to just update the current test to utilize minimum config. Imagine the user who only reads the first test for how to use the feature and gets the impression that additional config is required.